packages/devtools_app/lib/src/extensions/embedded/_view_web.dart

```diff
@@ -108,9 +108,9 @@ class _ExtensionIFrameController extends DisposableController
       }),
     );
 
-    html.window.addEventListener(
+    window.addEventListener(
       'message',
-      _handleMessageListener = _handleMessage,
+      _handleMessageListener = _handleMessage as EventListener,
```

Comment:
I believe this will fail since you're casting a Dart function to a JS function type. Prefer `_handleMessage.toJS as EventListener` instead.

As general guidance (until we get lints to ensure this!), avoid casts between Dart and JS interop types (casts between JS interop types are generally okay, granted you did the work to type check them or they are trivially true). They may work on one platform but not another.